script/elasticsearch_script.py

```diff
@@ -0,0 +1,45 @@
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk, scan
+import json
+
+es = Elasticsearch(
```

Comment:
Do we still need this script?

Reply:
Keep it for now just in case. This is to edit existing ones.